### ISSUE TITLE
Enhancement - Split Screen in the IDE

### ISSUE THREAD

**Issue by daba1208** [arduino/Arduino#8509]
It would be helpful if the Arduino IDE could have a "split-screen capability, so we can see two areas of code at the same time - pretty much the same as you can do in MS Excel..

I find mysel constantly scrolling up and down to see what I have named things, for example.  Having the ability to see 2 areas of code would be great.

**Comment by Nca78** [comment]
Getting highly frustrated with that too, ability to see 2 different places in the same file or two different files at the same time would save a lot of time.

**Comment by BAleiHi** [comment]
Yes, a horizontal split screen would be great! (Noob here - does admin prefer that we add a new comment if it's a repeat or just like the previous one?)  Thanks.
